esp/scm/ws_config2.ecm

```diff
@@ -310,6 +329,20 @@ ESPresponse [exceptions_inline] GetTreeResponse
 };
 
 
+ESPrequest FindNodesRequest
```

Comment:
It might be more common in hpcc code to call this FetchNodesRequest

Reply:
Unless there is a compelling reason, I will leave this as "find". A change to fetch will percolate all the way through the config manager.